samples/Http2SampleApp/Http2SampleApp.csproj

```diff
@@ -15,7 +15,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="../../test/shared/TestCertificates/testCert.pfx" CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="testCert.pfx">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Out of curiosity, does CopyToOutputDirectory need to be a tag, or does it still work as a attribute?

Reply:
It might work as an attribute but this is what VS consistently generates.

Reply:
But it looks cleaner as an attribute 😄